reference/forms/types/time.rst

```diff
@@ -129,6 +130,20 @@ on your underlying object. Valid values are:
 The value that comes back from the form will also be normalized back into
 this format.
 
+input_format
+~~~~~~~~~~~~
+
+**type**: ``string`` **default**: ``H:i:s``
+
+.. versionadded:: 4.3
+
+    The ``input_format`` option was introduced in Symfony 4.3.
+
+If the ``input`` option is set to ``string``, this option specifies the format
```

Comment:
I think we need to shorten this a bit as you cannot pass anything else then a string:

```rst
The ``input`` option specifies the format of the time. It must be a valid `PHP time format`_.
```